pages/guides/send-money/onboarding.mdx

```diff
@@ -100,19 +104,20 @@ print($customer); # => "https://api-sandbox.dwolla.com/customers/c7f300c0-f1ef-4
 ?>
 ```
 
-<ol class = "alerts">
-    <li class="alert icon-alert-alert">
-      Provide the IP address of the end-user accessing your application as the `ipAddress` parameter. This enhances fraud detection and tracking.
-    </li>
+<ol class="alerts">
+  <li class="alert icon-alert-alert">
+    Provide the IP address of the end-user accessing your application as the
+    `ipAddress` parameter. This enhances fraud detection and tracking.
+  </li>
 </ol>
 
 When the Customer is successfully created on your application, you will receive a `201` HTTP response with an empty response body. You can reference the Location header to retrieve a link that represents the created Customer resource. We recommend storing the full URL for future use, as it will be needed for actions such as attaching a bank or correlating webhooks that are triggered for the user in the Dwolla system.
 
 ## Step 1B. Handle Webhooks
 
-If you have an active [webhook subscription](https://developers.dwolla.com/guides/webhooks/create-subscription/subscribe-to-webhooks), you will receive the `customer_created` webhook immediately after the resource has been created.
+If you have an active [webhook subscription](/guides/webhooks/create-subscription/subscribe-to-webhooks), you will receive the `customer_created` webhook immediately after the resource has been created.
 
 <nav class="pager-nav">
```

Comment:
I think we can remove these `pager-nav`s now that we have the built-in pagination based on doc groups. Doesn't have to be in this ticket, just something to remember.

Reply:
Yep, planning to do that in the next large PR!